compiler/luci-interpreter/src/kernels/Relu6.h

```diff
@@ -40,8 +40,8 @@ class Relu6 : public Kernel
   void evalQuantized() const;
 
 private:
-  int32_t _output_multiplier;
-  int32_t _output_shift;
+  int32_t _output_multiplier = 0;
+  int32_t _output_shift = 0;
```

Comment:
Just curious, Relu.h is `{0}` and this is `= 0` ... hmmm... :)

Reply:
I fixed it too :-)